app/views/users/show.html.haml

```diff
@@ -0,0 +1 @@
+= link_to 'Edit your account', edit_user_registration_path
```

Comment:
I think you need to pass in an object to this path as well. like `current_user`....

Reply:
perhaps not....

Reply:
The link is only visible on a page that only the `current_user` can access, so I guess it works this way.